src/olympia/hero/tasks.py

```diff
@@ -0,0 +1,20 @@
+from olympia.hero.models import PrimaryHero
+from olympia.amo.celery import task
+from olympia.amo.decorators import use_primary_db
+
+@task()
```

Comment:
```suggestion
@task
```

(both should work but we use the plain syntax when we don't have arguments to pass)